src/Qowaiv/Diagnostics/Contracts/FluentSyntaxAttribute.cs

```diff
@@ -0,0 +1,8 @@
+namespace Qowaiv.Diagnostics.Contracts;
+
+/// <summary>To mark a method explicitly as impure. Methods decorated with
```

Comment:
Comment is for impure instead of FluentSyntaxAttribute.

Reply:
my bad. It is a special case of impure

Reply:
That is not true. It states that the return value is same instance as the input argument.